analyzer/evmtokenbalances/evm_token_balances.go

```diff
@@ -178,15 +214,12 @@ func (m Main) processStaleTokenBalance(ctx context.Context, batch *storage.Query
 			if balanceData.Balance.Cmp(staleTokenBalance.Balance) != 0 {
 				correction := &big.Int{}
 				correction.Sub(balanceData.Balance, staleTokenBalance.Balance)
-				// Note: This will happen because we currently don't scan
-				// before the beginning of the Dasmask upgrade, so the
-				// reckoning will be wrong about any balances from before
-				// then. It can also happen when contracts misbehave.
+				// Can happen when contracts misbehave, or if we haven't indexed all the runtime blocks from the very first one on.
 				m.logger.Warn("correcting reckoned balance of token to downloaded balance",
 					"token_addr", staleTokenBalance.TokenAddr,
 					"account_addr", staleTokenBalance.AccountAddr,
 					"download_round", staleTokenBalance.DownloadRound,
-					"reckoned_balance", staleTokenBalance.Balance,
+					"reckoned_balance", staleTokenBalance.Balance.String(),
```

Comment:
oh shoot did this break?

Reply:
IIRC I was seeing scientific notation in my logs

Reply:
oh it actually went in as a number? haha then yeah stringify